docs/data/data-grid/filtering-recipes/FilteredRowCount.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { DataGridPro, useGridApiRef, GridFilterModel } from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+
+const predefinedFilters: { label: string; filterModel: GridFilterModel }[] = [
+  {
+    label: 'All',
+    filterModel: { items: [] },
+  },
+  {
+    label: 'Filled',
+    filterModel: { items: [{ field: 'status', operator: 'is', value: 'Filled' }] },
+  },
+  {
+    label: 'Open',
+    filterModel: { items: [{ field: 'status', operator: 'is', value: 'Open' }] },
+  },
+  {
+    label: 'Rejected',
+    filterModel: { items: [{ field: 'status', operator: 'is', value: 'Rejected' }] },
+  },
+  {
+    label: 'Partially Filled',
+    filterModel: {
+      items: [{ field: 'status', operator: 'is', value: 'PartiallyFilled' }],
+    },
+  },
+];
+
+export default function FilteredRowCount() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 1000,
+    maxColumns: 10,
+  });
+
+  const apiRef = useGridApiRef();
+
+  const [predefinedFiltersRowCount, setPredefinedFiltersRowCount] = React.useState<
```

Comment:
I would probably prefer an object here, but I guess this is mainly personal preference. As long as the function is pure it should be fine.